contrib/Testing.mk

```diff
@@ -0,0 +1,135 @@
+# Version 0.1.7
+
+###############################################################################
+# Helpers for installing tooling..
+###############################################################################
+# Quick OS detection so we can pull down the right binaries
+OSNAME         ?= $(shell uname -s | tr A-Z a-z)
+ROOT_DIR       := $(shell git rev-parse --show-toplevel)
+BRANCH_NAME    := $(shell git rev-parse --abbrev-ref HEAD | sed 's/[_-]//g')
+BIN_DIR        ?= $(ROOT_DIR)/bin
+VENV_CMD       := python3 -m venv
+VENV_DIR       := $(ROOT_DIR)/.venv
+VENV_BIN       := $(VENV_DIR)/bin
+
+# This is the standare kind cluster-name that we use for local development.
+KIND_CLUSTER_NAME := default
+
+export PATH    := $(VENV_BIN):$(BIN_DIR):$(PATH)
+
+ifeq ($(OSNAME),linux)
+	GET_HELM_DOCS_URL ?= https://github.com/norwoodj/helm-docs/releases/download/v1.5.0/helm-docs_1.5.0_Linux_x86_64.tar.gz
+else
+	GET_HELM_DOCS_URL ?= https://github.com/norwoodj/helm-docs/releases/download/v1.5.0/helm-docs_1.5.0_Darwin_x86_64.tar.gz
+endif
+
+ifeq ($(HELM_TEST_ALL_CHARTS),true)
+	ALL_FLAG := --all
+endif
+
+HELM_VERSION   ?= v3.7.0
```

Comment:
Helm 3.7.0 was [released some time ago](https://github.com/helm/helm/releases/tag/v3.7.0). Would the latest version of Helm work too ?